app/user/token.py

```diff
@@ -0,0 +1,16 @@
+from typing import Any
+
+from django.contrib.auth.tokens import PasswordResetTokenGenerator
+from django.utils.encoding import smart_bytes
+
+
+class TokenGenerator(PasswordResetTokenGenerator):
+    def _make_hash_value(self, user: Any, timestamp: Any) -> Any:
+        return (
+            smart_bytes(user.pk)
+            + smart_bytes(timestamp)
```

Comment:
prefer to use format string over string concatenation

Reply:
I get an attribute error. Bytes object has no attribute format